lib/raygun.ts

```diff
@@ -24,6 +24,7 @@ import {
   SendOptions,
   Tag,
   Transport,
+  ApmBridge,
```

Comment:
Is there much of a performance hit now that this lives outside of the the project? 

I think this is much more cleaner than the previous implementation but just being wary of not having too much abstraction to avoid confusion :)

Reply:
> Is there much of a performance hit now that this lives outside of the the project?

I think on the contrary. The previous code tried to load the APM package, failed, caught an exception, and ignored it. That's unnecessary work that happens every time the Raygun4Node package would start.

Reply:
Ahh okay, this is all good then !